README.md

```diff
@@ -70,6 +73,8 @@ CGW_DB_PASS - PSQL DB password (credentials) to use upon connect to DB
 CGW_REDIS_DB_IP - IP of remote redis-db server to connect to
 CGW_REDIS_DB_PORT - PORT of remote redis-db server to connect to
 CGW_LOG_LEVEL - log level to start CGW application with (debug, info)
+CGW_CERTS_PATH - path to certificates located on host machine
+CGW_ALLOW_MISMATCH - allow client certificate CN and device MAC address mismatch (used for OWLS)
```

Comment:
I'd suggest to rename this to CGW_ALLOW_CERT_MISMATCH, just so it calls out that it's certificate related